paper.bib

```diff
@@ -77,7 +77,7 @@ @article{Hudson96
 @article{Holliday1996,
     author={Holliday, John D.
     and Willett, Peter},
-    title={Definitions of {\&}{\#}x201c;Dissimilarity{\&}{\#}x201d; for Dissimilarity-Based Compound Selection},
+    title={Definitions of "Dissimilarity" for Dissimilarity-Based Compound Selection},
```

Comment:
@gmrandazzo - The output includes unnecessary characters which should be rendered to quotes.

Due to the JOSS submisson: https://github.com/openjournals/joss-reviews/issues/5420

Reply:
Thanks! fixed with the commit https://github.com/gmrandazzo/libscientific/commit/ccc9cbc6afe9ac605416844f0444c8aa3744359f